overlord/devicestate/devicestate_install_api_test.go

```diff
@@ -338,7 +345,7 @@ func (s *deviceMgrInstallAPISuite) testInstallFinishStep(c *C, opts finishStepOp
 
 	s.state.Lock()
 	defer s.state.Unlock()
-	c.Check(chg.Err(), IsNil)
+	c.Assert(chg.Err(), IsNil)
```

Comment:
not related to this PR itself, but it seems we have quite a few tests in this package that could use this modification